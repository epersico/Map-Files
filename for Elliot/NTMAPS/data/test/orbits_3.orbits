# a:        0.9400000 , b:       0.2059000 
# omega:   (m =   8) / (n =   9) 
# brakl0:  -1.0000000 , braku0:   1.0000000 , brakn:           500
# acc:      1.000e-12 , lacc:       1.000e-08 , eqd:      1.000e-07
# a=  0.94000000000, b=  0.20590000000
#           q0             p0
    0.00000000    -0.42209075
   -0.22747097    -0.42209075
   -0.33224612    -0.21825018
   -0.39369325    -0.03923644
   -0.46102110     0.08829266
    0.46102110     0.13821735
    0.39369325     0.08829266
    0.33224612    -0.03923644
    0.22747097    -0.21825018    -1.2692e-12    -7.5444e-13

    0.00000000    -0.38544041
   -0.19965045    -0.38544041
   -0.29349810    -0.18975809
   -0.35356601     0.00849965
   -0.44147951     0.17232300
    0.44147951     0.24633695
    0.35356601     0.17232300
    0.29349810     0.00849965
    0.19965045    -0.18975809    -5.7107e-13    -3.4975e-13

    0.00000000    -0.20913019
   -0.10111131    -0.20913019
   -0.16821718    -0.08694503
   -0.23623644     0.09236408
   -0.37942931     0.29749463
    0.37942931     0.43898018
    0.23623644     0.29749463
    0.16821718     0.09236408
    0.10111131    -0.08694503    -8.8101e-14    -3.9254e-14

    0.00000000    -0.14285001
   -0.07918176    -0.14285001
   -0.14105038    -0.04458586
   -0.21346579     0.11492554
   -0.36698885     0.31542450
    0.36698885     0.46815881
    0.21346579     0.31542450
    0.14105038     0.11492554
    0.07918176    -0.04458586    -9.9045e-14    -1.0031e-14

    0.50000000     0.14285001
    0.42081824     0.14285001
    0.35894962     0.04458586
    0.28653421    -0.11492554
    0.13301115    -0.31542450
   -0.13301115    -0.46815881
   -0.28653421    -0.31542450
   -0.35894962    -0.11492554
   -0.42081824     0.04458586    -9.9045e-14     1.0031e-14

    0.50000000     0.20913019
    0.39888869     0.20913019
    0.33178282     0.08694503
    0.26376356    -0.09236408
    0.12057069    -0.29749463
   -0.12057069    -0.43898018
   -0.26376356    -0.29749463
   -0.33178282    -0.09236408
   -0.39888869     0.08694503    -8.8101e-14     3.9254e-14

    0.50000000     0.38544041
    0.30034955     0.38544041
    0.20650190     0.18975809
    0.14643399    -0.00849965
    0.05852049    -0.17232300
   -0.05852049    -0.24633695
   -0.14643399    -0.17232300
   -0.20650190    -0.00849965
   -0.30034955     0.18975809    -5.7107e-13     3.4976e-13

    0.50000000     0.42209075
    0.27252903     0.42209075
    0.16775388     0.21825018
    0.10630675     0.03923644
    0.03897890    -0.08829266
   -0.03897890    -0.13821735
   -0.10630675    -0.08829266
   -0.16775388     0.03923644
   -0.27252903     0.21825018    -1.2692e-12     7.5444e-13

    0.46102110     0.13821735
    0.39369325     0.08829266
    0.33224612    -0.03923644
    0.22747097    -0.21825018
   -0.00000000    -0.42209075
   -0.22747097    -0.42209075
   -0.33224612    -0.21825018
   -0.39369325    -0.03923644
   -0.46102110     0.08829266     4.5298e-01     1.0509e-13

    0.44147951     0.24633695
    0.35356601     0.17232300
    0.29349810     0.00849965
    0.19965045    -0.18975809
   -0.00000000    -0.38544041
   -0.19965045    -0.38544041
   -0.29349810    -0.18975809
   -0.35356601     0.00849965
   -0.44147951     0.17232300     4.2714e-01     1.7606e-14

    0.37942931     0.43898018
    0.23623644     0.29749463
    0.16821718     0.09236408
    0.10111131    -0.08694503
   -0.00000000    -0.20913019
   -0.10111131    -0.20913019
   -0.16821718    -0.08694503
   -0.23623644     0.09236408
   -0.37942931     0.29749463     3.5388e-01     7.7987e-15

    0.36698885     0.46815881
    0.21346579     0.31542450
    0.14105038     0.11492554
    0.07918176    -0.04458586
   -0.00000000    -0.14285001
   -0.07918176    -0.14285001
   -0.14105038    -0.04458586
   -0.21346579     0.11492554
   -0.36698885     0.31542450     3.3974e-01     2.6195e-14

   -0.13301115    -0.46815881
   -0.28653421    -0.31542450
   -0.35894962    -0.11492554
   -0.42081824     0.04458586
    0.50000000     0.14285001
    0.42081824     0.14285001
    0.35894962     0.04458586
    0.28653421    -0.11492554
    0.13301115    -0.31542450    -6.0576e-01    -2.6195e-14

   -0.12057069    -0.43898018
   -0.26376356    -0.29749463
   -0.33178282    -0.09236408
   -0.39888869     0.08694503
    0.50000000     0.20913019
    0.39888869     0.20913019
    0.33178282     0.08694503
    0.26376356    -0.09236408
    0.12057069    -0.29749463    -5.9502e-01    -7.7985e-15

   -0.05852049    -0.24633695
   -0.14643399    -0.17232300
   -0.20650190    -0.00849965
   -0.30034955     0.18975809
    0.50000000     0.38544041
    0.30034955     0.38544041
    0.20650190     0.18975809
    0.14643399    -0.00849965
    0.05852049    -0.17232300    -5.4418e-01    -1.7607e-14

   -0.03897890    -0.13821735
   -0.10630675    -0.08829266
   -0.16775388     0.03923644
   -0.27252903     0.21825018
    0.50000000     0.42209075
    0.27252903     0.42209075
    0.16775388     0.21825018
    0.10630675     0.03923644
    0.03897890    -0.08829266    -5.3093e-01    -1.0509e-13

