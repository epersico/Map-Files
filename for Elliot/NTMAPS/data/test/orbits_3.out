# a:        0.9400000 , b:       0.2059000 
# omega:   (m =   8) / (n =   9) 
# brakl0:  -1.0000000 , braku0:   1.0000000 , brakn:           500
# acc:      1.000e-12 , lacc:       1.000e-08 , eqd:      1.000e-07
#            x0              y0        res      yhalf   orb#  eq to   @ i=
# a=  0.94000000000, b=  0.20590000000
# shp=1 , lambdax=0.6192276      , lambday=0.6803942      
# shp=2 , lambdax=0.6526649      , lambday=0.6698343      
# shp=3 , lambdax=0.6192276      , lambday=0.6803942      
# shp=4 , lambdax=0.6526649      , lambday=0.6698343      
# shp=-1 , lambdax=0.6192276      , lambday=0.6652665      
# shp=-2 , lambdax=0.6126814      , lambday=0.6698343      
# shp=-3 , lambdax=0.6192276      , lambday=0.6652665      
# shp=-4 , lambdax=0.6126814      , lambday=0.6698343      
# sln=1 , maxbrack=4   , meql=4   
  0.00000000000  -0.42209075119     0.0575    0.08829      1      1      0
  0.00000000000  -0.38544041335    -0.0113    0.17232      2      2      0
  0.00000000000  -0.20913018797     0.0102    0.29749      3      3      0
  0.00000000000  -0.14285000747    -0.0297    0.31542      4      4      0

# sln=2 , maxbrack=4   , meql=4   
  0.50000000000   0.14285000747    -0.0297   -0.31542      5      5      0
  0.50000000000   0.20913018797     0.0102   -0.29749      6      6      0
  0.50000000000   0.38544041335    -0.0113   -0.17232      7      7      0
  0.50000000000   0.42209075119     0.0575   -0.08829      8      8      0

# sln=3 , maxbrack=4   , meql=4   
  0.46102110294   0.13821735164     0.0575   -0.42209      9      1      4
  0.44147951009   0.24633695099    -0.0113   -0.38544     10      2      4
  0.37942931025   0.43898017634     0.0102   -0.20913     11      3      4
  0.36698884625   0.46815880592    -0.0297   -0.14285     12      4      4

# sln=4 , maxbrack=4   , meql=4   
  0.86698884625  -0.46815880592    -0.0297    0.14285     13      5      4
  0.87942931025  -0.43898017634     0.0102    0.20913     14      6      4
  0.94147951009  -0.24633695099    -0.0113    0.38544     15      7      4
  0.96102110294  -0.13821735164     0.0575    0.42209     16      8      4

